### DISCUSSION TITLE
[WARN]  Ti.Ti.UI.iOS.SystemButtonStyle.BORDERED DEPRECATED in 10.0.0, in favor of Ti.Ti.UI.BUTTON_STYLE_TEXT / [WARN]  Ti.Ti.UI.iOS.SystemButtonStyle.DONE DEPRECATED in 10.0.0, in favor of Ti.Ti.UI.BUTTON_STYLE_OPTION_POSITIVE

### DISCUSSION THREAD

**Discussion by nicolomonili** [appcelerator/titanium_mobile#12971] [Q&A]
Ti.Ti.Ti.Ti.Ti.Ti.Ti.Ti.Ti.Ti...

```
[WARN]  Ti.Ti.UI.iOS.SystemButtonStyle.BORDERED DEPRECATED in 10.0.0, in favor of Ti.Ti.UI.BUTTON_STYLE_TEXT
[WARN]  Ti.Ti.UI.iOS.SystemButtonStyle.DONE DEPRECATED in 10.0.0, in favor of Ti.Ti.UI.BUTTON_STYLE_OPTION_POSITIVE
```

**SDK 10.0.0.GA**

**Comment by ewanharris** [comment]
This probably needs the strings fixing below, they need the `Ti.` prefix removing, happy to accept a PR if you're interested.

https://github.com/appcelerator/titanium_mobile/blob/fe3deddcea07ecb3555673cf2dba8d676fd79611/iphone/Classes/TiUIiOSSystemButtonStyleProxy.m#L19-L21

**Comment by m1ga** [comment]
Title and content could be improved next time :wink:  Looks like spam or an "auto issue" at a first quick view

**Comment by hbugdoll** [comment]
It's now obsolete: `TiUIiOSSystemButtonStyleProxy` was replaced by `TiUIiOSButtonConfigurationProxy` in https://github.com/tidev/titanium-sdk/pull/14252.
